plan/relations.go

```diff
@@ -212,6 +240,13 @@ func (n *NamedTableReadRel) CopyWithExpressionRewrite(rewriteFunc RewriteFunc, _
 	return &nt, nil
 }
 
+func (n *NamedTableReadRel) Remap(mapping ...int32) (Rel, error) {
+	newMapping, err := RemapHelper(n, mapping)
+	newRel := n
```

Comment:
Since this is a pointer receiver, I believe this isn't a copy, right?

Reply:
fixed.